### PR TITLE
12/avoid rest

### PR DESCRIPTION
**Pull request by jnduli**
Closes issue #12

### CODE REVIEW

File: pomodoro.sh [hunk 1]
```diff
@@ -32,8 +33,12 @@ count_down () {
     # Takes two parameters
     # $1 is time in minutes
     # $2 is messages to prepend
+    # $3 is n time it has looped spent (for continue support)
```
Comment (BonfaceKilz) [inline pomodoro.sh]: This comment is not clear

File: pomodoro.sh [hunk 2]
```diff
@@ -74,6 +83,7 @@ rest () {
 
 chiming_with_input () {
     echo "Press q to stop chiming, and start break"
+    echo "Press c to continue with task"
```
Comment (BonfaceKilz) [inline pomodoro.sh]: I'd recommend using heredocs like this:
```
cat <<EOF
Press q to stop chiming, and start break
Press c to continue with task

EOF
```

File: pomodoro.sh [hunk 3]
```diff
@@ -83,20 +94,32 @@ chiming_with_input () {
         duration=$SECONDS
         clear_line
         echo "Chiming duration: $((duration / 60)) min $((duration % 60)) sec"
-        if [[ $input = "q" ]] || [[ $input = "Q" ]]; then
-            echo
+        if [[ $input = "q" || $input = "Q" ]]; then
```
Comment (BonfaceKilz) [inline pomodoro.sh]: NIT:
```suggestion
        if [[ ${input^^} = "Q" ]]; then
```
`${input^^}`: the 2 carats capitalizes the variable

File: pomodoro.sh [hunk 4]
```diff
@@ -83,20 +94,32 @@ chiming_with_input () {
...
+            CONTINUE=false
+            break
+        fi
+        if [[ $input = "c" || $input = "C" ]]; then
```
Comment (BonfaceKilz) [inline pomodoro.sh]: Same as above.

**Review by BonfaceKilz — CHANGES_REQUESTED** [review]
I've run `shellcheck` and everything is ok. Other than that, check out the comments above.

**Comment by BonfaceKilz** [comment]
Also check this out: https://google.github.io/styleguide/shell.xml

**Comment by jnduli** [comment]
I've fixed  the above raised issues. Thanks for the suggestions.

File: pomodoro.sh [hunk 5]
```diff
@@ -82,8 +83,10 @@ rest () {
 }
 
 chiming_with_input () {
-    echo "Press q to stop chiming, and start break"
-    echo "Press c to continue with task"
+    cat <<EOF
+Press q to stop chiming and start break
+Press c to continue with task
+EOF
     echo ""
```
Comment (BonfaceKilz) [inline pomodoro.sh:90]: NIT: Since we are user heredocs with don't need this empty echo

**Review by BonfaceKilz — APPROVED** [review]
LGTM other than the minor nitpick on `echo ""` that I've pointed at above.

**Pull request merged**